src/func.cc

```diff
@@ -946,6 +946,29 @@ void ProfileFunc(const vector<Value*>& args, Evaluator* ev, string*) {
   }
 }
 
+void VariableLocationFunc(const vector<Value*>& args, Evaluator* ev, string* s) {
+  string arg = args[0]->Eval(ev);
+  WordWriter ww(s);
+  for (StringPiece var : WordScanner(arg)) {
```

Comment:
Why do you allow more than one argument here? Let's allow just one, at least for the time being?

Reply:
I ended up not using multiple values, but I think it's pretty reasonable. It fits with a bunch of the other make functions like $(dir), $(addsuffix), that operate on multiple values.

Reply:
I might have been mentally scarred by google3 and that's why I adopted the "if not absolutely necessary DISALLOW" stance.

However, since Kati is going away, I suppose this is OK?